Documentation/ceph-cluster-crd.md

```diff
@@ -153,6 +153,34 @@ spec:
 
 For more details, see the [Stretch Cluster design doc](https://github.com/rook/rook/blob/master/design/ceph/ceph-stretch-cluster.md).
 
+## Cluster on Multus
```

Comment:
These examples at the top of the doc are really focused on the different high level approaches to storage: raw devices, clusters on PVCs, and stretch clusters. I think the network example could apply to any of the storage examples, I think it's better to remove this high level example. We just need clarity in the network settings section.

Reply:
Can we put this in `Host-based Cluster` example?